src/movies.js

```diff
@@ -1,16 +1,63 @@
 // Iteration 1: All directors? - Get the array of all directors.
+
+function getAllDirectors(movies) {
+const result = movies.map(function(names) {
+    return names.director;
+})
+    return result;
+}
+
+
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors. How could you "clean" a bit this array and make it unified (without duplicates)?
 
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct?
 
+function howManyMovies(movies) {
+    const dramaMovies = movies.filter(function(number){
+        if(number.director === 'Steven Spielberg' && number.genre.indexOf('Drama') !== -1){
```

Comment:
Instead of using the if/else statement here, this condition can simply be returned:

`return (number.director === 'Steven Spielberg' && number.genre.indexOf('Drama') !== -1);`